roles/openshift_health_checker/openshift_checks/etcd_volume.py

```diff
@@ -0,0 +1,54 @@
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class EtcdVolume(OpenShiftCheck):
+    """Ensures etcd storage usage does not exceed a given threshold."""
+
+    name = "etcd_volume"
+    tags = ["etcd", "health"]
+
+    default_etcd_device_usage_threshold_percent = 90
+    # where to find ectd data, higher priority first.
+    supported_mount_paths = ["/var/lib/etcd", "/var/lib", "/var", "/"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        # TODO: only execute this check on hosts in the 'ectd' group?
+        # Maybe also 'masters' if there are no standalone etcd hosts?
```

Comment:
Yes, we should do this, as in the logging stack check.

A quick read of https://docs.openshift.org/latest/install_config/install/advanced_install.html#single-master seems to indicate that etcd hosts will always be defined, but I just ran a single-master install without any and it worked fine.

I think maybe the logic should be something like:

    etcd_hosts = get_var(task_vars, "groups", "etcd", default=[]) or get_var(task_vars, "groups", "masters", default=[]) or []
    if get_var(task_vars, "ansible_ssh_host") in etcd_hosts:
        ...

This PR is not the place, but we could think a little about generalizing this logic of which subset of hosts a check runs against. I think it's likely this is only the place where a task does something like this (as opposed to the play that includes it).